packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -20,52 +20,43 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
 
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
 
-  const checkmarkFontColor = semanticColors.inputForegroundChecked;
-  // TODO: after updating the semanticColors slots mapping this needs to be semanticColors.inputBorder
-  const checkmarkFontColorHovered = palette.neutralSecondary;
-  // TODO: after updating the semanticColors slots mapping this needs to be semanticColors.smallInputBorder
-  const checkboxBorderColor = palette.neutralPrimary;
-  const checkboxBorderIndeterminateColor = semanticColors.inputBackgroundChecked;
-  const checkboxBorderColorChecked = semanticColors.inputBackgroundChecked;
-  const checkboxBorderColorDisabled = semanticColors.disabledBodySubtext;
-  const checkboxBorderHoveredColor = semanticColors.inputBorderHovered;
-  const checkboxBorderIndeterminateHoveredColor = semanticColors.inputBackgroundCheckedHovered;
-  const checkboxBackgroundChecked = semanticColors.inputBackgroundChecked;
-  // TODO: after updating the semanticColors slots mapping following 2 tokens need to be semanticColors.inputBackgroundCheckedHovered
-  const checkboxBackgroundCheckedHovered = semanticColors.inputBackgroundCheckedHovered;
-  const checkboxBorderColorCheckedHovered = semanticColors.inputBackgroundCheckedHovered;
-  const checkboxHoveredTextColor = semanticColors.inputTextHovered;
-  const checkboxBackgroundDisabledChecked = semanticColors.disabledBodySubtext;
-  const checkboxTextColor = semanticColors.bodyText;
-  const checkboxTextColorDisabled = semanticColors.disabledText;
+  const checkmarkColor = {
+    font: semanticColors.inputForegroundChecked,
+    // TODO: after updating the semanticColors slots mapping this needs to be semanticColors.inputBorder
+    fontHovered: palette.neutralSecondary
+  };
+
+  const checkboxColor = {
+    // TODO: after updating the semanticColors slots mapping this needs to be semanticColors.smallInputBorder
+    border: palette.neutralPrimary,
+    borderIndeterminate: semanticColors.inputBackgroundChecked,
+    borderIndeterminateHovered: semanticColors.inputBackgroundCheckedHovered,
+    borderChecked: semanticColors.inputBackgroundChecked,
+    borderHovered: semanticColors.inputBorderHovered,
+    borderDisabled: semanticColors.disabledBodySubtext,
+    backgroundChecked: semanticColors.inputBackgroundChecked,
+    backgroundCheckedHovered: semanticColors.inputBackgroundCheckedHovered,
+    borderCheckedHovered: semanticColors.inputBackgroundCheckedHovered,
+    backgroundDisabledChecked: semanticColors.disabledBodySubtext,
+    text: semanticColors.bodyText,
+    textHovered: semanticColors.inputTextHovered,
+    textDisabled: semanticColors.disabledText
+  };

```

Comment:
Hmm, size auditor results are saying size actually went up by 322 bytes. This structure may actually work against you. I suspect references to `checkboxColor.backgroundCheckedHovered` may only reduced to `c.backgroundCheckedHovered`. May be worth trying the original approach to confirm.

Reply:
Interesting, I calculated I removed around 700 characters. I'll revert it like you say to see if there's a difference

Reply:
There's not always a direct correlation between what you remove and what can be minified. The object and variable names can be minified, but the key names cannot. You removed a bunch of variable names that can be minified, but then added a bunch of references to keys, which can't be minified. 😄

Reply:
As an example:

Before: 
`checkboxBorderIndeterminateColor ` could reduce to `c`

After:
`checkboxColor.borderIndeterminate` can only reduce to `c.borderIndeterminate` because key names can't be minified

Reply:
Oh! Great explanation, that makes 100% sense and look at that we dropped the size down -258 bytes!